### PR TITLE
Create config.yml

### PR DESCRIPTION
**Pull request by faithngetich**
Issue [link](https://github.com/publiclab/plots2/issues/3681#issuecomment-429254015)

### CODE REVIEW

**Comment by faithngetich** [comment]
@jywarren I have accepted the changes you request but I have a concern. The link returns a 404 error when I try to access it. Do you have an idea why this is the case?

File: .github/config.yml [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+# Comment to be posted to on PRs from first time contributors in your repository
+newPRWelcomeComment: |
+ Thanks for opening this pull request! `Dangerbot` will test out your code and reply in a bit with some pointers and requests.
+ There may be some errors, **but don't worry!** We're here to help! 👍🎉😄
+# Configuration for first-pr-merge - https://github.com/behaviorbot/first-pr-merge
+
+# Comment to be posted to on pull requests merged by a first time user
+firstPRMergeComment: |
+ Congrats on merging your first pull request! 🙌🎉⚡️ 
+ Your code will likely be published to https://publiclab.github.io/leaflet-environmental-layers/ in the next few days.
```
Comment (jywarren) [inline .github/config.yml]:
```suggestion
 Your code will likely be published to https://publiclab.github.io/leaflet-environmental-layers/example/ in the next few days. 
```

**Comment by faithngetich** [comment]
@jywarren done.

**Comment by jywarren** [comment]
Awesome, thanks!!!

**Pull request merged**